floor/floor/controller/midi/functions.py

```diff
@@ -26,64 +27,94 @@ def add(cls, name, **kwargs):
 
 
 MidiFunctions.add('playlist_next',
+                  callback=lambda controller, _: controller.playlist.advance(1),
```

Comment:
Oops, my psuedocode had a bug - drop the `(1)` here and line 33; should be `controller.playlist.advance()`.

Reply:
Right.